Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -36,6 +38,8 @@
 
 NSString *const BTPayPalDriverErrorDomain = @"com.braintreepayments.BTPayPalDriverErrorDomain";
 
+NSString *const BTPayPalNativeCheckoutErrorDomain = @"com.braintreepayments.BTPayPalNativeCheckoutErrorDomain";
```

Comment:
Not sure if we want to use a new error domain, or add more specific error codes in `BTPayPalDriverErrorType` specific to Native XO 🤔

Reply:
This is fair - I thought about this and kept going back and forth. I'll change these to use the already existing error domain.